Hot Deals Section Tests

Vladislav2498 opened the pull request.

Vladislav2498 committed:
Hot Deals Section Tests

Ahmed-Fahmy-Dev commented on client/src/pages/HomePage/HomePage.js:
```diff
@@ -1,114 +1,114 @@
+            <AboutUkraine/>
+        </Box>
+        <Box component="section">
+            {/*<CardContainer />*/}
```
this is not corrent.. please leave it as it is as i already wrote the homepage test and mocked this component.

Ahmed-Fahmy-Dev commented on client/src/components/CardContainer/CardContainer.test.js:
```diff
@@ -1,9 +1,13 @@
+        const {asFragment} = render(<CardContainer/>, {wrapper: RouterWrapper});
         expect(asFragment()).toMatchSnapshot();
     });
 });
```
please mock the MUI components of typography
you can try to mock Swiper too

Ahmed-Fahmy-Dev reviewed (commented):
please check Eslint errors:

![image](https://user-images.githubusercontent.com/107735322/200577399-bbb10429-1f28-4ede-b8fd-e5a97d47a6d0.png)

Vladislav2498 committed:
hot-deals tests hotfixes

Ahmed-Fahmy-Dev reviewed (changes requested):
not approved.    
CardItem.test.js  is not complete.   MUI components have to be mocked there too

Ahmed-Fahmy-Dev commented:
ESlint errors fixed by myself.

Ahmed-Fahmy-Dev committed:
hotfix: fixed ESlint errors
- added eslint rule for whole file
- imported memoryRouter before CardContainer

Vladislav2498 committed:
Merge branch 'develop' into hotDealsTests

Vladislav2498 committed:
Hot Deals Section Tests fixed

Ahmed-Fahmy-Dev committed:
hotfix: fixed test file and snapshot
- changed mocked Box to img tag
- updated snapshot

Ahmed-Fahmy-Dev reviewed (approved):
approved. 

i added a hot fix by myself.. please check commit message for details

Ahmed-Fahmy-Dev committed:
Merge branch 'develop' into hotDealsTests
